SIL.Archiving/SIL.Archiving.csproj

```diff
@@ -34,7 +34,7 @@ See full changelog at https://github.com/sillsdev/libpalaso/blob/master/CHANGELO
     <PackageReference Include="GitVersionTask" Version="5.3.4">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="L10NSharp" Version="4.1.0-beta0036" />
+    <PackageReference Include="L10NSharp" Version="*-*" />
```

Comment:
I think it would be better to specify a minimum version so that we have some documentation of a version that successfully built, i.e. 4.1.0*-*.

(For reference mainly for myself: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning)

Reply:
good idea; done